lib/v2/nasa/apod-cn.js

```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 10;
+    const rootUrl = `https://www.nasachina.cn/wp-json/wp/v2/posts?categories=2&per_page=${limit}`;
+    const { data } = await got({
+        method: 'get',
+        url: rootUrl,
+    });
+
+    const items = await Promise.all(
+        data.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: item.link,
```

Comment:
No need to visit `item.link`. The full article is already included in the /posts request(content.rendered).